cmd/zstream/zstream.c

```diff
@@ -49,6 +60,9 @@ zstream_usage(void)
 int
 main(int argc, char *argv[])
 {
+	if (argc >= 1 && strendswith(argv[0], "zstreamdump"))
+		return (zstream_do_dump(argc, argv));
```

Comment:
I think this would match "foozstreamdump", which I don't think is what we want.  Maybe using `basename()` would help simplify this?  Something like `strcmp(basename(argv[0], "zstreamdump"))`?

Reply:
I mean it doesn't matter too much, but sure; updated with effectively that code.